chain/actors/actor_cron.go

```diff
@@ -1,48 +1,9 @@
 package actors
 
 import (
-	"github.com/filecoin-project/go-address"
-	"github.com/filecoin-project/lotus/chain/actors/aerrors"
-	"github.com/filecoin-project/lotus/chain/types"
+	"github.com/filecoin-project/specs-actors/actors/builtin/cron"
 )
 
-type CronActor struct{}
+type CronActor = cron.Actor
```

Comment:
Can we do it without type aliases (reference spec-actors directly)?

Reply:
yes, but i think we can just ship this now and clean things later

Reply:
SGWM